packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -88,6 +99,13 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		6801C8332555D726009DAF8D /* Frameworks */ = {
```

Comment:
This is the default Frameworks build phase that Xcode adds for every new target.  This one is empty, so it could be deleted, but most people keep the default ones.